### DISCUSSION TITLE
Can (SumatraPDF-settings.txt) be constant between releases and ordered better?

### DISCUSSION THREAD

**Discussion by ImSpecial** [sumatrapdfreader/sumatrapdf#2680] [General]
An example of what I mean:

A fresh 3.4.5 settings file;
https://pastebin.com/raw/d6J8JV9A

and IMO, how it should look;
https://pastebin.com/raw/Gam312gg

This would help with upgrades and keeping settings better organized/better readability. Is something like this possible? Right now everything seems to be in random orders, even just looking at my previous not so old v3.3.3 file, it's much different in the ordering.

**Comment by kjk** [comment]
Makes sense, opened https://github.com/sumatrapdfreader/sumatrapdf/issues/2681 to track this.

Except for `FileState` and `SessionData` and other settings that are not supposed to be edited. They'll be at the end.
